src/LuaInterface.cpp

```diff
@@ -71,17 +71,17 @@ QStringList LuaInterface::varName(TVar* var)
         return names;
     }
     names << var->getName();
-    TVar* p = var->getParent();
-    while (p && p->getName() != "_G") {
-        names.insert(0, p->getName());
-        p = p->getParent();
+    TVar* parent = var->getParent();
+    while (parent && parent->getName() != "_G") {
+        names.insert(0, parent->getName());
+        parent = parent->getParent();
     }
     return names;
 }
 
-bool LuaInterface::validMove(QTreeWidgetItem* p)
+bool LuaInterface::validMove(QTreeWidgetItem* widgetItem)
 {
-    TVar* newParent = varUnit->getWVar(p);
+    TVar* newParent = varUnit->getWVar(widgetItem);
```

Comment:
```suggestion
    auto pNewParent = varUnit->getWVar(pWidget);
```
with `newParent` being renamed through the method.